carLot/invLoad.js

```diff
@@ -0,0 +1,27 @@
+CarLot = (function (carLot) {
+
+  carLot.populateCarLot = function(inventory) {
+
+    carHolder = document.getElementById("carHolder")
+
+  	carsArray = inventory[0].cars;
+
+    for(var i = 0; i < carsArray.length; i++){
+      carHolder.innerHTML += `
+        <div class="car" style="border-color : ${carsArray[i].Color}">
```

Comment:
Loop over your array of cars and build up an HTML string to build a card for each car. Also, use Bootstrap to create rows. Each row should contain 3 columns. Make sure you have a parent element with a class of container.